src/components/ICA/ICA.js

```diff
@@ -146,15 +188,28 @@ ICA.propTypes = {
 
   /** Specify whether or not the values should be truncated. */
   truncate: PropTypes.bool,
+
+  /** The size of the ICA
+   * @type string
+   */
+  size: PropTypes.oneOf(['default', 'large', 'xlarge']),
```

Comment:
For consistency, could we align with [Carbon's naming for sizes](https://react.carbondesignsystem.com/?path=/docs/components-button--default#button-size)? For example, `lg` instead of `large`.